networking/metallb/metallb-upgrading-operator.adoc

```diff
@@ -6,18 +6,18 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-The automatic upgrade procedure does not work as expected from {product-title} 4.10 and earlier. A summary of the upgrade procedure is as follows:
+The automatic upgrade procedure does not work as expected from {product-title} 4.10 and earlier. Upgrading from any version greater than {product-title} 4.10 to the latest version works as expected. A summary of the upgrade procedure from {product-title} 4.10 and earlier is as follows:
```

Comment:
PTAL the IBM Style Guide's section on [Product versions](https://www.ibm.com/docs/en/ibm-style?topic=references-product-versions). I think there are some useful patterns that would help make this clearer. Note, use _later_ to refer to versions instead of _greater than_.